quickwit/Cargo.toml

```diff
@@ -36,9 +36,38 @@ members = [
   "quickwit-telemetry",
 ]
 default-members = [
+  "quickwit-actors",
+  "quickwit-aws",
   "quickwit-cli",
+  "quickwit-cluster",
+  "quickwit-codegen",
+  "quickwit-codegen/example",
+  "quickwit-common",
+  "quickwit-config",
+  "quickwit-control-plane",
+  "quickwit-index-management",
+  "quickwit-datetime",
+  "quickwit-directories",
+  "quickwit-doc-mapper",
   "quickwit-indexing",
-  "quickwit-metastore"
+  "quickwit-ingest",
+  "quickwit-integration-tests",
+  "quickwit-jaeger",
+  "quickwit-janitor",
+  "quickwit-macros",
```

Comment:
I guess we don't want `quickwit-lambda` here, but maybe put it commented to be explicit?

Reply:
Yes, Lambda tests are not run in CI. @fmassot can we enable them now?

Reply:
I would avoid that for now.